index.html

```diff
@@ -6,17 +6,12 @@
   <meta charset="UTF-8">
   <title>Fimento API</title>
   <link rel="icon" type="image/png" href="./dist/assets/favicon.png">
-  <link rel="stylesheet" type="text/css" href="./dist/css/global.css" />
+  <!-- <link rel="stylesheet" type="text/css" href="./dist/css/global.css" /> -->
 </head>
 
 <body>
-  <header class="fimento-header">
-    <a href="https://www.fimento.com/">
-      <img src="./dist/assets/fimento_logo.svg" />
-    </a>
-  </header>
   <div>
-    <redoc spec-url='./specifications.yml'></redoc>
+    <redoc></redoc>
```

Comment:
What is this? Do we need this empty tags? If we do it should probably just be `<redoc/>` as well

Reply:
that is true, since now it is invoked through javascript instead of HTML

Reply:
Actually it still need the html tag so I will change to <redoc/>